apps/innbyggers-flate/src/pages/UtkastPage.tsx

```diff
@@ -0,0 +1,128 @@
+import { BodyLong, Button, Detail, GuidePanel, Heading, List, Link, Alert } from '@navikt/ds-react'
+import { ThumbUpIcon } from '@navikt/aksel-icons'
+import { svg } from '../ikoner/nav-veileder'
+import { EMDASH, formatDateStrWithMonthName, hentTiltakNavnHosArrangørTekst } from '../utils/utils'
+import { useParams } from 'react-router-dom'
+import { DeferredFetchState, useDeferredFetch } from '../hooks/useDeferredFetch'
+import { godkjennUtkast } from '../api/api'
+import { VilIkkeGodkjenneExpansionCard } from '../components/VilIkkeGodkjenneExpansionCard'
+import { useDeltakerContext } from '../DeltakerContext'
+import { Tiltakstype } from '../api/data/deltaker'
+
+export const UtkastPage = () => {
+  const { deltaker, setDeltaker } = useDeltakerContext()
+  const arrangorNavn = deltaker.deltakerliste.arrangorNavn
+  const dagerIUkatext = deltaker.dagerPerUke
+    ? `${deltaker.dagerPerUke} ${deltaker.dagerPerUke > 1 ? 'dager' : 'dag'} i uka`
+    : ''
+
+  const { deltakerId } = useParams()
+  const { state, error, doFetch: doFetchGodkjennUtkast } = useDeferredFetch(godkjennUtkast)
+  const handleGodkjennUtkast = () => {
+    if (deltakerId) {
+      doFetchGodkjennUtkast(deltakerId).then((oppdatertDeltaker) => {
+        if (oppdatertDeltaker) setDeltaker(oppdatertDeltaker)
+      })
+    }
+  }
+
+  return (
+    <div className="flex flex-col items-start">
+      <Heading level="1" size="large">
+        Utkast til påmelding
+      </Heading>
+      {deltaker.vedtaksinformasjon?.sistEndret && (
+        <div className="flex gap-2 mt-2">
+          <Detail weight="semibold">Sist endret:</Detail>
+          <Detail>
+            {formatDateStrWithMonthName(deltaker.vedtaksinformasjon?.sistEndret)}{' '}
+            {deltaker.vedtaksinformasjon?.sistEndretAv}
+          </Detail>
+        </div>
+      )}
+
+      <GuidePanel poster illustration={svg} className="mt-4">
+        <Heading level="2" size="small">
+          Dette er et utkast til påmelding til{' '}
+          {hentTiltakNavnHosArrangørTekst(deltaker.deltakerliste.tiltakstype, arrangorNavn)}
+        </Heading>
+        <BodyLong className="mt-2">
+          Før vi sender dette til {arrangorNavn} vil vi gjerne at du leser gjennom. Hvis du
+          godkjenner utkastet blir du meldt på, vedtak fattes og {arrangorNavn} mottar informasjon.
+        </BodyLong>
+      </GuidePanel>
+
+      <Heading level="2" size="medium" className="mt-6">
+        Hva er innholdet?
+      </Heading>
+      <BodyLong size="small" className="mt-2">
+        Du får tett oppfølging og støtte av en veileder. Sammen kartlegger dere hvordan din
```

Comment:
Jeg tror kanskje denne teksten er `ledetekst` fra `deltakelsesinnholdSchema` som du får med deltakeren? I så fall er det kanskje lurt å bruke den i stedet for å hardkode det inn :)

Reply:
ah! supert!